receiver/mongodbreceiver/scraper_test.go

```diff
@@ -382,3 +382,61 @@ func TestTopMetricsAggregation(t *testing.T) {
 		require.EqualValues(t, expectedCommandValues, actualOperationTimeValues["commands"])
 	})
 }
+
+func TestServerAddressAndPort(t *testing.T) {
+	tests := []struct {
+		name            string
+		serverStatus    bson.M
+		expectedAddress string
+		expectedPort    int64
+		expectedErr     error
+	}{
+		{
+			name: "address_only",
+			serverStatus: bson.M{
+				"host": "localhost",
+			},
+			expectedAddress: "localhost",
+			expectedPort:    27017,
```

Comment:
```suggestion
			expectedPort:    defaultMongodbPort,
```
Nit: Use central variable